compiler/rustc_codegen_cranelift/src/intrinsics/mod.rs

```diff
@@ -439,6 +439,12 @@ fn codegen_regular_intrinsic_call<'tcx>(
 
             ret.write_cvalue(fx, a);
         }
+        sym::is_val_statically_known => {
+            intrinsic_args!(fx, args => (_a); intrinsic);
+
+            let res = fx.bcx.ins().iconst(cranelift_codegen::ir::types::I8, 0);
```

Comment:
This can be shortened as

```suggestion
            let res = fx.bcx.ins().iconst(types::I8, 0);
```

as `cranelift_codegen::ir::types` is already imported. Other than the cg_clif changes LGTM.

Reply:
I'll wait to see if I get more suggested changes before I squash.